src/components/SystemPages/NewReaderControls.vue

```diff
@@ -0,0 +1,377 @@
+        isValidPageSet: {{isValidPageSet}}
+      </pre> -->
+
+      <div v-if="promptMspaMode" class="settings application" >
+        <dl>
+          <template v-for="boolSetting in settingListBoolean">
+            <dt :key="boolSetting.model"><label>
+              <input type="checkbox" 
+                :name="boolSetting.model" 
+                :id="boolSetting.model" 
+                v-model="$localData.settings[boolSetting.model]"
+              >{{boolSetting.label}}</label></dt> 
+              <!-- the spacing here is made of glass -->
+            <label :for="boolSetting.model">
+              <dd class="settingDesc" v-html="boolSetting.desc" />
+            </label>
+          </template>
+        </dl>
+      </div>
+    </template>
+
+    <div v-if="featureList.includes('fastforward')" class="settings application" >
+      <h3>Reading Experience</h3>
+      <dl class="fastForwardSelection">
+        <dt>
+          <input type="radio" id="fast_forward=false" :value="false" 
+            v-model="myFastForward"/>
+          <label for="fast_forward=false">Replay</label>
+        </dt>
+        <dd><label for="fast_forward=false">Read as if you were reading it live.<br>
+          All pages will be presented how they were as of the time of your most recent page. (with some minor exceptions; see 
+          <!-- Don't link to CC in setup mode (no settings page yet!) -->
+          <template v-if="false && $localData.assetDir">
+            <a href='/settings/controversial'>controversial content</a>).
+          </template>
+          <template v-else>
+            "controversial content" in Settings).
+          </template>
+        </label></dd>
+
+        <dt>
+          <input type="radio" id="fast_forward=true" :value="true" 
+            v-model="myFastForward"/>
+          <label for="fast_forward=true">Archival</label>
+        </dt>
+        <dd><label for="fast_forward=true">Read as an archival reader.<br>
+          Stories will be presented approximately as they were at the time they were finished (or abandoned).</label></dd>
+      </dl>
+    </div>
+  </div>
+</template>
+
+<script>
+import StoryPageLink from '@/components/UIElements/StoryPageLink.vue'
+
+export default {
+  name: 'NewReaderControls',
+  emits: ['enable', 'disable', 'change', 'test', 'ffchange'],
+  components: {StoryPageLink},
+  props: {
+    promptMspaMode: {
+      type: Boolean,
+      default: true
+    },
+    features: {
+      type: String,
+      default: "pagenumber fastforward"
+    },
+    forceGateChoice: {
+      default: false
+    },
+    handleEnable: {},
+    handleDisable: {},
+    handleChange: {}
+  },
+  data: function() {
+    return {
+      // The number in the input field. May be an mspa number or viz number depending on settings. Mutable.
+      // TODO: This assignment sometimes doesn't work, and assigns 001901 anyway?
+      newReaderPageInput: this.$newReaderCurrent || this.$mspaOrVizNumber('001901'),
```

Comment:
$newReaderCurrent defaults to 001901 out of a factory reset, then gets parsed as a viz number. Not ideal!

Reply:
Bug, fixed